tfjs-backend-wasm/starter/parcel/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "@tensorflow/tfjs-backend-wasm": "3.14.0",
-    "@tensorflow/tfjs-core": "3.14.0"
+    "@tensorflow/tfjs-core": "4.4.0"
```

Comment:
We shouldn't set core and wasm backend in different major versions.

Reply:
Oh, that should not have happened. Nice catch! I'll take a look.